packages/sql-libsql/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@effect/sql-libsql",
+  "version": "0.1.0",
+  "type": "module",
+  "license": "MIT",
+  "description": "A libSQL toolkit for Effect",
+  "homepage": "https://effect.website",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Effect-TS/effect.git",
+    "directory": "packages/sql-libsql"
+  },
+  "bugs": {
+    "url": "https://github.com/Effect-TS/effect/issues"
+  },
+  "tags": [
+    "typescript",
+    "sql",
+    "libsql",
+    "database"
+  ],
+  "keywords": [
+    "typescript",
+    "sql",
+    "libsql",
+    "database"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist",
+    "provenance": true
+  },
+  "scripts": {
+    "codegen": "build-utils prepare-v2",
+    "build": "pnpm build-esm && pnpm build-annotate && pnpm build-cjs && build-utils pack-v2",
+    "build-esm": "tsc -b tsconfig.build.json",
+    "build-cjs": "babel build/esm --plugins @babel/transform-export-namespace-from --plugins @babel/transform-modules-commonjs --out-dir build/cjs --source-maps",
+    "build-annotate": "babel build/esm --plugins annotate-pure-calls --out-dir build/esm --source-maps",
+    "check": "tsc -b tsconfig.json",
+    "test": "vitest",
+    "coverage": "vitest --coverage"
+  },
+  "devDependencies": {
```

Comment:
You can remove /platform here, as you don't have a Migrator module.

For migrations, do you use some tooling from turso?

Reply:
Hmm after some research I think we need to include a Migrator module too.

Reply:
Yeah, I was just trying to add the minimum to start with.